application/__init__.py

```diff
@@ -19,7 +19,8 @@ def create_app(config_name):
         SWAGGER_URL,
         API_URL,
         config={
-            'app_name': "auctionable_change_api"
+            'app_name': "auctionable_change_api",
+            'defaultModelsExpandDepth': -1
         }
```

Comment:
This hides the "Schemas" section at the bottom of Swagger page

Reply:
nice deep-dive detail . . .